bibliopixel/remote/trigger_process.py

```diff
@@ -0,0 +1,10 @@
+from .. util import importer
```

Comment:
Can't... the process method has to be in it's own module to work on Windows because... Windows. And yes... I'm keeping the process for now :)